api/utils/revisions.go

```diff
@@ -70,17 +70,20 @@ func ParseRevision(revision string, allowNext bool) (*Revision, error) {
 	return &Revision{uint32(n)}, err
 }
 
-// GetSummary returns the block summary for the given revision,
-// revision required to be a deterministic block other than "next".
-func GetSummary(rev *Revision, repo *chain.Repository, bft bft.Committer) (sum *chain.BlockSummary, err error) {
+func ParseNumberRevision(revision uint32) *Revision {
+	return &Revision{revision}
+}
+
+func ParseBlockID(rev *Revision, repo *chain.Repository, bft bft.Committer) (thor.Bytes32, error) {
```

Comment:
I was checking the usages of this, and perhaps it might be possible to get away without this new method.

Reply:
I am not sure, this logic was internal to `GetSummary`. Since the newest block will be in the new cache, the revision has to be parsed somehow.

Reply:
This is done now, back to the previous version